### DISCUSSION TITLE
[Enhancement] Add another sponsor method

### DISCUSSION THREAD

**Discussion by superbonaci** [mucommander/mucommander#1016] [General]
### Description

The sponsorship page forces me to fill certain details which I don't provide, so what about other methods like PayPal, etc.?
https://github.com/sponsors/mucommander

**Comment by ahadas** [comment]
@superbonaci what are those details that you are asked to provide?

**Reply by superbonaci** [reply]
Such information should be detailed here:
https://docs.github.com/sponsors

Which unfortunately is not listed.

**Reply by ahadas** [reply]
sorry, I don't understand yet - what is the information you were asked to provide that other methods don't require?

**Reply by maiemi** [reply]
@ahadas maybe the data of the credit card. I am unhappy, too, to be forced to provide these data. But because of your excellent work, I bit the bullet :)

**Reply by ahadas** [reply]
ack
I see that there's [a thread on this](https://github.com/orgs/community/discussions/45180#) and it was suggested there to check [the integration between GitHub and Patreon](https://docs.github.com/en/sponsors/sponsoring-open-source-contributors/sponsoring-an-open-source-contributor-through-patreon) for sponsoring with PayPal so maybe it is worth a try (please share your feedback if that works/doesn't work for you). I'd like to use the limited time I can work on muCommander these days for other things that are unrelated to sponsorship, but if this still remains an issue we can check alternatives later on
